libs/ledger-live-common/src/exchange/swap/getExchangeRates.ts

```diff
@@ -46,12 +46,12 @@ const getExchangeRates: GetExchangeRates = async (
   const dexProviders = ["paraswap", "oneinch"];
```

Comment:
Instead have this array, can we move this logic to the provider definition?
```
.filter((provider) =>
      provider.type === "DEX" ? includeDEX : true
  )

Reply:
I've added the type to `swap/index.ts`, so I can change it to this `getProviderConfig(provider.provider).type === "DEX" ? includeDEX : true`